include/srtp.h

```diff
@@ -1424,7 +1408,7 @@ typedef enum {
 
 typedef struct srtp_event_data_t {
   srtp_t        session;  /**< The session in which the event happend. */
-  srtp_stream_t stream;   /**< The stream in which the event happend.  */
+  uintptr_t     ssrc;     /**< The ssrc in host order of the stream in which the event happend */
```

Comment:
I think the use of `uintptr_t` requires some additional checks and defines configure and integers.h akin to what we do in order to support other C99ish types.
According to C99 7.18.1.4 the type is also optional so I think we need some guards, even for C99 compliant compilers.